confgenerator/testdata/valid/linux/all-backward_compatible_with_explicit_exporters/golden_e59640aecc0edba3f991865e880afb46.lua

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Sync'd offline. For cases where a LUA script is shared across multiple pipelines, prefixing the `md5sum` with `{pipeline_id}.{RECEIVER_ID}` would cause duplication and complexity.

Given that tradeoff, we decided to just bear with it for the debugging / troubleshooting inconvenience.

Hopefully this would become moot when https://github.com/fluent/fluent-bit/pull/5302 lands.